scripts/translations.py

```diff
@@ -0,0 +1,98 @@
+import argparse
+from upload import upload_task
+
+parser = argparse.ArgumentParser()
+parser.add_argument('-f', '--force', dest='force', action="store_true", default=False)
+args = parser.parse_args()
+
+languages = [
+    ('Simplified Chinese'),
+    ('German'),
+    ('Russian'),
+    ('Italian'),
+    ('French'),
+    ('Turkish'),
+    ('Portuguese'),
+    ('Arabic'),
+    ('Hindi'),
+]
```

Comment:
It would also be great to add a task that allows the student to propose a language on chat.zulip.org, and if we approve it, they can do it (with the idea that we'd approve basically anything).